### DISCUSSION TITLE
Price transparency IRF

### DISCUSSION THREAD

**Discussion by kloughnane** [CMSgov/hospital-price-transparency#74] [Q&A] [answered]
Can we use CMGs for the price transparency file?

**Comment by RidenourK** [comment]
@kloughnane Thank you for your question. Can you please clarify what are you referring to by "CMGs"?

**Reply by cascrim** [reply]
@kloughnane   I am not sure but HIPPS code may meet your needs.

Inpatient Rehabilitation Facility Prospective Payment System   Under the inpatient rehabilitation facility prospective payment system (IRF PPS), a casemix adjusted payment for varying numbers of days of IRF care is made using Case Mix Groups (CMG). On Medicare claims these CMGs are represented as HIPPS codes.  

The following scheme has been developed to create distinct 5-position, alphanumeric IRF HIPPS codes:  The first position of the code represents a comorbidity tier.  Comorbidities that may appear in the case of an IRF patient are arrayed in three tiers based on whether the costs associated with that comorbidity are considered high, medium or low.   The first position of the IRF PPS HIPPS codes will only allow alphabetical characters.  The valid values for this position are as follows: A = without comorbidities B = comorbidity in tier 1 (high) C = comorbidity in tier 2 (medium) D = comorbidity in tier 3 (low) The second, third, fourth and fifth positions of the code represent the CMG itself.

https://www.cms.gov/medicare/medicare-fee-for-service-payment/prospmedicarefeesvcpmtgen/downloads/hippsuses.pdf

**Comment by kloughnane** [comment]
Inpatient rehab hospitals are not paid on DRGs, they are paid based on CMGs (Case Mix Group).  We wouldn’t be able to list the DRGs because we don’t bill or get paid that way.

I think CMG needs to be added to this list below as a valid value for rehab hospitals.

Standard Name
Reporting Value
Current Procedural Terminology
CPT
National Drug Code
NDC
Healthcare Common Procedural Coding System
HCPCS
Revenue Code
RC
International Classification of Diseases
ICD
Diagnosis Related Groups
DRG
Medicare Severity Diagnosis Related Groups
MS-DRG
Refined Diagnosis Related Groups
R-DRG
Severity Diagnosis Related Groups
S-DRG
All Patient, Severity-Adjusted Diagnosis Related Groups
APS-DRG
All Patient Diagnosis Related Groups
AP-DRG
All Patient Refined Diagnosis Related Groups
APR-DRG
Ambulatory Payment Classifications
APC
Local Code Processing
LOCAL
Enhanced Ambulatory Patient Grouping
EAPG
Health Insurance Prospective Payment System
HIPPS
Current Dental Terminology
CDT
Charge Description Master (chargemaster)
CDM
TriCare Diagnosis Related Groups
TRIS-DRG

**Comment by CarlsonErin** [comment, marked as answer]
[@kloughnane](url) Thanks for your feedback.     In the CMS standard template version 2.0 CMG is not a valid code type value.  In the standard template documentation, we offer the following guidance:  The value "LOCAL" may be used for internal accounting codes in conjunction with another billing code for that item or service. However, if no other code types are available for a particular item or service, "LOCAL" may be used as a valid value.
